include/exadg/poisson/user_interface/application_base.h

```diff
@@ -59,6 +60,7 @@ class ApplicationBase
   add_parameters(dealii::ParameterHandler & prm)
   {
     output_parameters.add_parameters(prm);
+    grid_parameters.add_parameters(prm);
```

Comment:
The solver `poisson/solver.h` contains the option to create a default input file, see the function `create_input_file(std::string const & input_file)`. If I remember correctly, the parameters are added to the input file in the order they are added in the code. Hence, if we want `grid_parameters` as an "input" to occur before "output" variables, we should probably change the order here. Note that the order is also different below when defining grid and output parameters as class members.

Reply:
Yes you are right. I will look into this! I will also change the order in `input.json`, if needed.